killFlask.sh

```diff
@@ -1,6 +1,3 @@
 #!/bin/bash
 
-#FLASK_APP_FILE="flask_init.py"
-#kill -9 `ps aux | grep $FLASK_APP_FILE | grep -v grep | awk '{ print $2 }'`
-
 kill -9 `ps aux | grep flask | grep -v grep | awk '{ print $2 }'`
```

Comment:
still need a new line at the end of the file

Reply:
I could swear I put it there. Will bee added next PR.
BTW, Indeed all code tested, and sudo never prompted for password.